CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ConversationExtensions.cs

```diff
@@ -88,5 +193,69 @@ public static IEnumerable<TeamsChannelAccount> AsTeamsChannelAccounts(this IEnum
                 yield return JObject.FromObject(channelAccount).ToObject<TeamsChannelAccount>();
             }
         }
+
+        private static async Task<Tuple<HttpRequestMessage, HttpResponseMessage>> SetupAndSendHttpRequestGetConversationMember(ConnectorClient Client, string conversationId, string userId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            // Construct URL
+            var _baseUrl = Client.BaseUri.AbsoluteUri;
+            var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "v3/conversations/{conversationId}/members/{userId}").ToString();
+            _url = _url.Replace("{conversationId}", System.Uri.EscapeDataString(conversationId));
+            _url = _url.Replace("{userId}", System.Uri.EscapeDataString(userId));
```

Comment:
do we have tests covering this URL construction ?

Reply:
Ooh good point; will add some